srcpkgs/ruby-erubi/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'ruby-erubi'
+pkgname=ruby-erubi
+version=1.12.0
+revision=1
+build_style=gem
+short_desc="Erubi is a ERB template engine for ruby"
+maintainer="Orphaned <orphan@voidlinux.org>"
```

Comment:
packages shouldn't be orphaned to start

Reply:
Added myself as the maintainer of the package.